src/assets/style.css

```diff
@@ -113,6 +113,11 @@ body {
 
 a {
   color: var(--accent-color);
+
+  &:hover,
+  &:focus {
+    text-decoration: none;
+  }
```

Comment:
I'm a bit uncomfortable with using CSS nesting so soon. caniuse says 81% of global support. It's hard to trust this % of course, but it's sufficiently low, and the browser market is sufficiently fragmented, that we're bound to have users who don't have support for the feature yet.
I know it's looking pretty green on the usual top 4. But I'm not sure about Opera, Samsung Internet and UCBrowser (caniuse and BCD seem to disagree).

Reply:
removed all nesting!